corehq/apps/user_importer/validation.py

```diff
@@ -132,6 +133,17 @@ def validate_spec(self, spec):
             return self.error_message
 
 
+class TableauRoleValidator(ImportValidator):
```

Comment:
cool, I'll add that

Reply:
While working on tableau group validation I realized that tableau groups are appended to the same row so that when you download web users and grab the value for `tableau_groups` you get a string like "city,state". 
This is different from how phone numbers are handled for mobile workers where a new column is added for each phone number, so `phone-number 1` `phone-number 2` etc
https://github.com/dimagi/commcare-hq/blob/38139efc13404a1598250c0ebb20480c54a3f2cb/corehq/apps/users/bulk_download.py#L88
So this will require a little reworking so that get_tableau_group_ids_by_names is called with a list of group names and not a series of single chars. 
https://github.com/dimagi/commcare-hq/blob/38139efc13404a1598250c0ebb20480c54a3f2cb/corehq/apps/reports/util.py#L494
Curious to hear thoughts on converting to an approach like with `phone-numbers` where a new column is created for each tableau group vs keeping multiple tableau groups in the same column and splitting on the ",".

Reply:
Yeah, that's a fair point. This has ballooned a bit as it is. I'll make a ticket for the backlog and handle tableau_groups as they are now.

Reply:
So I had already written 99% of group validation. It just needed to split out the individual groups. I'll make a ticket for reworking the columns, though.